src/java.base/share/classes/java/lang/System.java

```diff
@@ -1816,8 +1816,8 @@ private static void initPhase1() {
         // stdout/err.encoding are set when the VM is associated with the terminal,
         // thus they are equivalent to Console.charset(), otherwise the encodings
         // of those properties default to native.encoding
-        setOut0(newPrintStream(fdOut, props.getProperty("stdout.encoding")));
-        initialErr = newPrintStream(fdErr, props.getProperty("stderr.encoding"));
+        setOut0(newPrintStream(fdOut, StaticProperty.stdoutEncoding()));
+        initialErr = newPrintStream(fdErr, StaticProperty.stderrEncoding());
```

Comment:
This is initPhase1 where we've just saved the initial properties. I think it would be better to not change this. It would be okay to use tempProps here too, but I think clearer as is.